examples/with-msw/.env.production

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_MOCKING=enabled
```

Comment:
Hey, @jensmeindertsma. Was it intentional to enable MSW in production?

I understand the idea is to have MSW running on preview builds but I can hardly find a use case for that. Shouldn't the preview build acts as an integration environment, implying you're previewing your feature against the _actual_ API before merging and, thus, releasing it to production?

There's been a discussion opened (https://github.com/mswjs/msw/discussions/895) that having MSW enabled in production by default may be confusing. MSW is close-to-never intended to run in production (except if you're building an educational website, perhaps), so having it enabled in this example is misleading.